azure-pipelines.yaml

```diff
@@ -39,35 +39,21 @@ stages:
       displayName: 'Install NodeJS and yarn'
       condition: eq(variables['imageName'], 'macOS-12')
 
-    - script: |
-        sudo apt-get install curl
-        curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
-        sudo apt-get install nodejs
-      displayName: 'Install NodeJS'
-      condition: eq(variables['imageName'], 'ubuntu-20.04')
-
-    - script: |
-        curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-        echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-        sudo apt update && sudo apt install yarn
-      displayName: 'Install yarn'
-      condition: eq(variables['imageName'], 'ubuntu-20.04')
-
-    - script: python -m pip install --upgrade ipywidgets traitlets pytest build jupyter-packaging==0.10.6 jupyterlab==3.2.3
+    - script: python -m pip install --upgrade ipywidgets traitlets pytest build jupyter-packaging==0.10.6 jupyterlab==3.6.5
```

Comment:
Do we want a test matrix to include jupyterlab 4?

Reply:
Good idea!